src/components/NavBar/NavBar.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react'
+import { Link, useLocation } from 'react-router-dom'
+import './NavBar.css'
+
+function NavBar() {
+  const location = useLocation()
+
+  const initialShowAllDrinks = location.pathname === '/all'
+  const [showAllDrinks, setShowAllDrinks] = useState(initialShowAllDrinks)
+
+  const toggleDrinks = () => {
+    setShowAllDrinks(!showAllDrinks)
+  }
+
+  return (
+    <div>
+    <div className="border"></div>
+      <nav className="nav-container">
+        <ul className="nav-list">
+          <li className="nav-item">
+            <Link to="/random">Random Drink</Link>
+          </li>
+          <li className="nav-item" onClick={toggleDrinks}>
+            <Link to={showAllDrinks ? '/na' : '/all'}>
+              {showAllDrinks ? 'NA Drinks' : 'All Drinks'}
+            </Link>
+          </li>
+          <li className="nav-item">
+            <Link to="/favorites">Favorite Drinks</Link>
+          </li>
+        </ul>
+      </nav>
+        <div className="border"></div>
```

Comment:
indentation is off. It also looks like you have a `div` wrapped around another `div`? Is there a reason you have two of them there?